### PR TITLE
Minor improvements to chat, cheating & console

### PR DESCRIPTION
**Pull request by rollerozxa**
Just some minor things

### CODE REVIEW

File: content/chat.md [hunk 1]
```diff
@@ -6,150 +6,113 @@ aliases:
...
 ```
 
-
 ### Server messages
-
 Server messages are messages that are sent from the server, not from a player. The format is:
 
-```
+```txt
 Server -!- message
```
Comment (grorp) [inline content/chat.md:106]: I doubt these exist anymore, couldn't find them by grepping in minetest/minetest

Comment (wsor4035) [reply content/chat.md:106]: there is a countdown in chat when your restart a server with a time, not sure what format that uses iirc

Comment (rollerozxa) [reply content/chat.md:106]: Doesn't seem to be used a lot but still a thing apparently

image

Comment (rollerozxa) [reply content/chat.md:106]: I think this section can be cut though. Mods can use whatever format for global server messages and the timed shutdown uses "***" to signify a server message.

Comment (rollerozxa) [reply content/chat.md]: Actually I rewrote it a bit instead.

**Pull request merged**